orbit-2-livedata/README.md

```diff
@@ -0,0 +1,47 @@
+# Orbit 2 Live Data plugin
+
+The Live Data plugin provides:
+
+- [Container](../orbit-2-core/src/main/java/com/babylon/orbit2/Container.kt)
+  extensions for LiveData streams.
+
+## Including the module
+
+```kotlin
+implementation("com.babylon.orbit2:orbit-livedata:<latest-version>")
+```
```

Comment:
again mention registration

Reply:
live data needs no registration. Only DSL plugins do so coroutines and rxjava currently

Reply:
I got over-excited adding comments in :)